modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -131,7 +131,7 @@
 	for high-ranking Cybersun officers, a few spares have been reserved for field agents."
 	item = /obj/item/radio/headset/chameleon/advanced
 	cost = 4 //Also a BIGBOY tool. Though inconvienent to wield, this allows the wearer to spy and interact with any one frequency they desire, even without the proper encryption key, along with flashbang protection and loudmode. Cannot breach syndiecomms by itself.
-	progression_minimum = 20 MINUTES
+	progression_minimum = 60 MINUTES
```

Comment:
I think utility items such as this can remain at ten, since they aren't gonna cause much issue